[16.0][MIG] stock_picking_return_refund_option: Migration to version 16.0

Yadier-Tecnativa opened the pull request:
@Tecnativa TT42442

sergio-teruel committed:
[11.0][NEW] stock_picking_return_refund_option: New module for set refund options in picking directly

sergio-teruel committed:
[MIG] stock_picking_return_refund_option: Migration to v12.0

sergio-teruel committed:
[IMP] stock_picking_return_refund_option: black, isort

sergio-teruel committed:
[MIG] stock_picking_return_refund_option: Migration to v13.0

sergio-teruel committed:
[MIG] stock_picking_return_refund_option: Migration to v13.0

rousseldenis committed:
[13.0][FIX] stock_picking_return_refund_option: Fix dependencies

sergio-teruel committed:
[13.0][FIX] stock_picking_return_refund_option: Compute delivered quantities when user check to refund in operation lines.

sergio-teruel committed:
[FIX] stock_picking_return_refund_option: Use Form in tests to avoid crash test due to this Odoo commit https://github.com/OCA/OCB/commit/8ca10a8f39c6e6b8647a695aa543f008f01ba034

pedrobaeza committed:
[IMP] stock_picking_return_refund_option: Make tests resilient

We force the delivery creating stock.move.line for avoiding
circumstances where the assignation is not done.

The SO creation is also simplified using Form utility, and now as we
don't depend on the warehouse, we remove such code as well.

OCA-git-bot committed:
stock_picking_return_refund_option 13.0.1.0.1

pedrobaeza committed:
[IMP] stock_picking_return_refund_option: More resilience in tests

There's still some cases that makes the pickings to not be completed, so
we force them and also switch to Form for simplifying code.

OCA-git-bot committed:
stock_picking_return_refund_option 13.0.1.0.2

cesar-tecnativa committed:
[MIG] stock_picking_return_refund_option: Migration to 15.0

[UPD] Update stock_picking_return_refund_option.pot

[UPD] README.rst

Added translation using Weblate (Catalan)

Translated using Weblate (Catalan)

Currently translated at 42.8% (3 of 7 strings)

Translation: account-invoicing-15.0/account-invoicing-15.0-stock_picking_return_refund_option
Translate-URL: https://translation.odoo-community.org/projects/account-invoicing-15-0/account-invoicing-15-0-stock_picking_return_refund_option/ca/

MiquelRForgeFlow committed:
[FIX] stock_picking_return_refund_option: deprecated SavepointCase class

OCA-git-bot committed:
stock_picking_return_refund_option 15.0.1.0.1

Yadier-Tecnativa committed:
[MIG] stock_picking_return_refund_option: pre-commit stuff

fix

pedrobaeza commented:
/ocabot migration stock_picking_return_refund_option

pedrobaeza commented on stock_picking_return_refund_option/__manifest__.py:
```diff
@@ -3,7 +3,7 @@
     "name": "Stock Picking Return Refund Option",
     "summary": "Update the refund options in pickings",
-    "version": "15.0.1.0.1",
+    "version": "16.0.1.0.1",
```
```suggestion
    "version": "16.0.1.0.0",
```

Yadier-Tecnativa replied:
Changed

pedrobaeza reviewed (commented):
And please put the setup changes separated in the pre-commit stuff commit (see the migration guide)

Yadier-Tecnativa committed:
[MIG] stock_picking_return_refund_option: Migration to 16.0

Change reference from move_lines to move_ids
The field move_lines in Odoo 15 change to move_ids in Odoo 16

fix mig - 16

chienandalu reviewed (approved):
Code and functional :+1:

pedrobaeza commented:
/ocabot merge nobump

sergio-teruel commented:
ping @rafaelbn

sergio-teruel commented:
Done! 
@juangomezzurita
